src/lib.rs

```diff
@@ -44,7 +44,7 @@ pub use sqlx_mysql::{self as mysql, MySql, MySqlConnection, MySqlExecutor, MySql
 #[doc(inline)]
 pub use sqlx_postgres::{self as postgres, PgConnection, PgExecutor, PgPool, Postgres};
 
-#[cfg(feature = "sqlite")]
+#[cfg(any(feature = "sqlite", feature = "sqlite-unbundled"))]
 #[cfg_attr(docsrs, doc(cfg(feature = "sqlite")))]
```

Comment:
This needs to match the above `#[cfg]` for it to be reflected in the docs:

```suggestion
#[cfg_attr(docsrs, doc(cfg(any(feature = "sqlite", feature = "sqlite-unbundled"))))]
```